auth.html

```diff
@@ -11,7 +11,7 @@
       href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
       rel="stylesheet"
     />
-    <link rel="stylesheet" href="http://maxui.netlify.app/components/max.css" />
+    <link rel="stylesheet" href="https://maxui.netlify.app/components/max.css" />
```

Comment:
Define these links into the root directory and import the root directory here.

Reply:
Can you please explain how is that different from adding this here?

Reply:
If you use N no of files for CSS in larger projects there might be a chance you can forget to add one or two.  
By defining these links to the root directory or any other file it will import by default into your project and the chances of errors by missing files are reduced.